<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopShareClass</class>
 <widget class="QWidget" name="DesktopShareClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DesktopShare</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background: #000000;
}
QLabel, QRadioButton, QSpinBox, QPushButton,QSpinBox,
QLineEdit{
	color: #fefefe;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>15</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #191b26;
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="OpenglScreenPaint" name="openGLWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background: green;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_6" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background: #3c404b;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #2b2e38;
border-radius: 10px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>麦克风</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QSlider" name="microphone_slider">
               <property name="value">
                <number>99</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="microphone_label">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="desktop_share.qrc">:/DesktopShare/res/voice-on.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="topMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>桌面音频</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QSlider" name="speaker_slider">
               <property name="value">
                <number>99</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="speaker_label">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="desktop_share.qrc">:/DesktopShare/res/voice-on.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget {
background: #2b2e38;
border-radius: 10px;
}
QPushButton {
	background: #3c404b;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="ClickableLabel" name="rtmp_push_btn">
           <property name="styleSheet">
            <string notr="true">QLabel {
background: #3c404b;
}
QLabel:hover {
    background-color: #4f535e;    /* 鼠标悬停时的背景颜色 */
}</string>
           </property>
           <property name="text">
            <string>RTMP 推流</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ClickableLabel" name="rtsp_push_btn">
           <property name="styleSheet">
            <string notr="true">QLabel {
	background: #3c404b;
}
QLabel:hover {
    background-color: #4f535e;    /* 鼠标悬停时的背景颜色 */
}</string>
           </property>
           <property name="text">
            <string>RTSP 推流</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ClickableLabel" name="recording_btn">
           <property name="styleSheet">
            <string notr="true">QLabel {
	background: #3c404b;
}
QLabel:hover {
    background-color: #4f535e;    /* 鼠标悬停时的背景颜色 */
}</string>
           </property>
           <property name="text">
            <string>开始录制</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ClickableLabel" name="setting_btn">
           <property name="styleSheet">
            <string notr="true">QLabel {
	background: #3c404b;
}
QLabel:hover {
    background-color: #4f535e;    /* 鼠标悬停时的背景颜色 */
}</string>
           </property>
           <property name="text">
            <string>设置</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ClickableLabel" name="exit_btn">
           <property name="styleSheet">
            <string notr="true">QLabel {
	background: #3c404b;
}
QLabel:hover {
    background-color: #4f535e;    /* 鼠标悬停时的背景颜色 */
}</string>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OpenglScreenPaint</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">opengl_screen_paint.h</header>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickable_label.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="desktop_share.qrc"/>
 </resources>
 <connections/>
</ui>
